crt_portal/cts_forms/templates/forms/confirmation.html

```diff
@@ -135,10 +135,10 @@ <h3 class="intake-section-title">
             {% trans "Legal aid offices or members of lawyer associations in your state may be able to help you with your issue." %}"
             <ul class="confirmation-list">
               <li>
-                {% trans 'American Bar Association, visit <a href="http://www.findlegalhelp.org">www.findlegalhelp.org</a> or call <a href="tel:800-285-2221">(800) 285-2221</a>' %}
+                {% trans 'American Bar Association, visit <a class="external-link" aria-label="www.findlegalhelp.org" href="http://www.findlegalhelp.org/">www.findlegalhelp.org</a> or call <a href="tel:800-285-2221">(800) 285-2221</a>' %}
               </li>
               <li>
-                {% trans 'Legal Service Corporation (or Legal Aid Offices), visit <a href="https://www.lsc.gov/find-legal-aid">www.lsc.gov/find-legal-aid</a> or call <a href="tel:202-295-1500">(202) 295-1500</a>' %}
```

Comment:
lsc.gov/find-legal-aid redirects to https://www.lsc.gov/what-legal-aid/find-legal-aid are we sure we don't want to send people staight there?

Reply:
Good question. Looks like there's a 301 redirect. Note that we might need to change this url in two places -- confirmation and landing. What do you want to do here?